lms/urls.py

```diff
@@ -33,6 +33,11 @@
     url(r'', include('student.urls')),
     # TODO: Move lms specific student views out of common code
     url(r'^dashboard$', 'student.views.dashboard', name="dashboard"),
+
+    url(r'^course/{}/refund_status$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
@clintonb The purpose is to get refund status for a course to show some specific information on learner dashboard. For now this change is specific to dashboard only and would be the only place we would be using this . If we want to make it generic for other clients/apps, then we can create another ticket to move this to a more appropriate place like you suggested, and let this requested change not be a road block for this PR. thanks